Code/Main/shared/src/main/scala/rescala/reactives/SchedulerException.scala

```diff
@@ -0,0 +1,21 @@
+package rescala.reactives
+
+import rescala.core.{Derived, ReSource}
+import rescala.extra.simpleprop.SimpleStruct
+
+sealed trait SchedulerException extends RuntimeException {
```

Comment:
The InvariantViolationException seems to be the only subclass of SchedulerException? So probably the SchedulerException is not needed?

Reply:
Done in https://github.com/rescala-lang/REScala/pull/32/commits/9795e498b497179887e000b2546b8f51b742cbec